universal-application-tool-0.0.1/app/repository/QuestionRepository.java

```diff
@@ -33,6 +33,57 @@ public QuestionRepository(EbeanConfig ebeanConfig, DatabaseExecutionContext exec
         executionContext);
   }
 
+  static class PathConflictDetector {
+    Boolean conflict = Boolean.FALSE;
+    String newPath;
```

Comment:
Suggestion (blocking): `private final` for `newPath`, `private` for `conflict`

Reply:
Done.